dev/release/post-09-docs.sh

```diff
@@ -42,20 +42,20 @@ popd
 pushd "${ARROW_DIR}"
 git checkout "${release_tag}"
 
-docker-compose build ubuntu-cpp
-docker-compose build ubuntu-python
-docker-compose build ubuntu-docs
-docker-compose run --rm \
+archery docker run \
   -v "${ARROW_SITE_DIR}/docs:/build/docs" \
   -e ARROW_DOCS_VERSION="${version}" \
   ubuntu-docs
-popd
 
-pushd "${ARROW_SITE_DIR}"
-git add docs
-git commit -m "[Website] Update documentations for ${version}"
-git push -u origin ${branch_name}
-popd
+: ${PUSH:=1}
+
+if [ ${PUSH} -gt 0 ]; then
+  pushd "${ARROW_SITE_DIR}"
+  git add docs
+  git commit -m "[Website] Update documentations for ${version}"
+  git push -u origin ${branch_name}
+  popd
+fi
```

Comment:
Could you also put the last `echo "Success!" ...` block into this `if ...fi`?
If we don't push, the message is meaningless.

Could define `github_url` that is used at the last line? `github_url` isn't defined in this file. (Or we can replace `${github_url}` with `https://github.com/apache/arrow`.)
(This is not related to this change.)

Reply:
done